test/ejabberd_tests/test.config

```diff
@@ -31,6 +31,10 @@
                  {domain, <<"localhost">>},
                  {vars, "node2_vars.config"},
                  {cluster, mim}]},
+         {mim3, [{node, ejabberd3@localhost},
```

Comment:
I'd love to name it with `mongooseim` prefix. I feel free to rename also `ejabberd2@localhost` node to `mongooseim2@localhost`. Is there any reason why not to do that?